src/simplewallet/simplewallet.cpp

```diff
@@ -2730,7 +2787,7 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
         if (m_restore_multisig_wallet)
         {
             const char *prompt = "Specify multisig seed: ";
-            m_electrum_seed = input_line(prompt);
+            m_electrum_seed = input_secure_line(prompt);
```

Comment:
Maybe the same is applicable to the wallet restoration from secret keys, like L2912?
```c++
      std::string spendkey_string = input_line("Secret spend key: ");
```

Reply:
Yes, various other places.